### DISCUSSION TITLE
default bin plugin format is different than the exported

### DISCUSSION THREAD

**Discussion by gustavosuarez** [sagamusix/JDTools#12] [General]
First of all,  thanks a lot for this great tool. Looking into the libraries that come with zenology pro plugin I noticed that binary format is different. Header starts with 'KoaBankFile..' value. Have you ever seen this before?
Gus

**Comment by sagamusix** [comment]
I know that this identifier is being used for D-50 and TB-303 BIN files, among others. Roland uses the .bin file extension across most of their plugins, but the underlying file format differs. I have not found any KoaBankFile-type files that would be compatible with the JD-800 model, though. Did you find any KoaBankFile BINs where you are sure they belong to the JD-800 model?

**Comment by gustavosuarez** [comment]
That KoaBankFile is present in all the libraries I have downloaded from the Roland Cloud. All the models share that bin KoaBankFile. BUT once you export a patch or library, the plugin writes in svz format.
We can talk offline and I can share all my patches to you (pretty sure that there are more than 4k patches), and we can research.

**Reply by sagamusix** [reply]
Shoot me a mail via https://sagamusix.de/en/e-mail/

**Comment by sagamusix** [comment]
For anyone wondering about the outcome of this thread, the files were indeed not intended for the JD-800 model, thus this format remains unsupported by JDTools.
